src/Microsoft.TestPlatform.CrossPlatEngine/Discovery/DiscovererEnumerator.cs

```diff
@@ -118,7 +118,6 @@ public void LoadTests(IDictionary<string, IEnumerable<string>> testExtensionSour
     /// </summary>
     /// <param name="extensionAssembly"> The extension Assembly. </param>
     /// <param name="sources"> The sources.   </param>
-    /// <param name="settings"> The settings.   </param>
```

Comment:
This seems wrong, `settings` parameter does exist.

Reply:
the name was incorrect